sqlmesh/core/model/decorator.py

```diff
@@ -18,16 +24,26 @@
 logger = logging.getLogger(__name__)
 
 
+DECORATOR_RETURN_TYPE = t.TypeVar("DECORATOR_RETURN_TYPE")
```

Comment:
We shouldn't redefine this here, it's already available in `sqlmesh.core.model`.